spec/views/hyrax/vector_works/show.html.erb_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe "hyrax/vector_works/show.html.erb", vector_partial: true do
+  let(:resource) do
+    FactoryGirl.build(:vector_work, id: 'test',
+                                    title: ['a vector work'],
+                                    creator: ['Dangermond'],
+                                    date_created: ['1969-12-31'],
+                                    rights: ['In Copyright'])
+  end
+  let(:solr_document) { SolrDocument.new(resource.to_solr) }
+  let(:ability) do
+    a = double("ability")
+    allow(a).to receive(:can?).and_return(true)
+    a
+  end
+  let(:presenter) { VectorWorkShowPresenter.new(solr_document, ability) }
+  let(:blacklight_config) { CatalogController.new.blacklight_config }
+  let(:editor) { true }
+  let(:collector) { true }
+  before do
+    stub_blacklight_views
+    assign(:presenter, presenter)
+    expect(view).to receive(:provide).with(:page_title, 'a vector work')
+    expect(view).to receive(:provide).with(:page_header).and_yield
```

Comment:
No apologies, this concern is warranted.  I specifically based this off of the tests in `hyrax` 1.0.1:
https://github.com/samvera/hyrax/blob/v1.0.1/spec/views/hyrax/base/edit.html.erb_spec.rb#L21

I will address these and update the PR.